src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -93,6 +98,43 @@ def view(wf_run_id: t.Optional[str], config: t.Optional[str]):
     action.on_cmd_call(wf_run_id, config)
 
 
+DOWNLOAD_DIR_OPTION = cloup.option(
+    "--download-dir",
+    help=(
+        "Directory path to store the artifact value. If passed, the command will "
+        "create a file under this location."
+    ),
+    type=click.Path(file_okay=False, dir_okay=True, writable=True, path_type=Path),
+)
+
+
+@workflow.command()
+@cloup.argument("wf_run_id", required=False)
+@cloup.option("-c", "--config")
+@DOWNLOAD_DIR_OPTION
+def results(
+    wf_run_id: t.Optional[str],
+    config: t.Optional[str],
+    download_dir: t.Optional[Path],
+):
+    """
+    Shows preview of a workflow output values corresponding to the variables
+    returned from the ``@workflow`` function. This tries to print a human-friendly
+    values preview, but the output isn't guaranteed to be a valid parseable value. If
+    you need the artifact value for further processing, use the ``download_dir`` option
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏We use `artifact` as the outputs of each task, atm.

wdyt about:
```suggestion
    you need the result value for further processing, use the ``download_dir`` option
```